bus-mapping/src/evm/opcodes/pop.rs

```diff
@@ -0,0 +1,123 @@
+use core::ops::Deref;
+use super::Opcode;
+use crate::{
+    exec_trace::{ExecutionStep, TraceContext},
+    operation::{StackOp, RW},
+    Error,
+};
+
+/// Placeholder structure used to implement [`Opcode`] trait over it corresponding to the
+/// [`OpcodeId::POP`](crate::evm::OpcodeId::POP) `OpcodeId`.
+/// This is responsible of generating all of the associated [`StackOp`]s and place them
+/// inside the trace's [`OperationContainer`](crate::operation::OperationContainer).
+#[derive(Debug, Copy, Clone)]
+pub(crate) struct Pop;
+
+impl Opcode for Pop {
+    fn gen_associated_ops(
+        &self,
+        ctx: &mut TraceContext,
+        // Contains the POP instr and the value which is last placed in the stack.
+        exec_step: &mut ExecutionStep,
+        // Contains the next step.
+        next_steps: &[ExecutionStep],
+    ) -> Result<(), Error> {
+        ctx.push_op(
+            exec_step,
+            StackOp::new(
+                RW::READ,
+                // Get the value and addr from the execution step and reads the value of the last filled stack element
+                exec_step.stack().last_filled(),
+                exec_step.
+                    .stack()
+                    .deref()
+                    .last()
+                    .cloned()
+                    .ok_or(Error::InvalidStackPointer)?,
+            ),
+        );
+
+        Ok(())
+    }
+}
+
+#[cfg(test)]
+mod pop_tests {
+    use super::*;
+    use crate::{
+        bytecode,
+        evm::{
+            EvmWord, GasCost, GasInfo, OpcodeId, Stack, StackAddress, Storage,
+        },
+        external_tracer, BlockConstants, ExecutionTrace,
+    };
+    use pasta_curves::pallas::Scalar;
+
+    #[test]
+    fn pop_opcode_impl() -> Result<(), Error> {
+        let code = bytecode! {
+            #[start]
```

Comment:
Since you don't want to test the `PUSH1` you should place the `#[start]` label before it. On that way, it will execute and load the value in the stack but not appear in the trace.